### PR TITLE
Refactor some code in everything.js file to make code more clear and DRY

### PR DESCRIPTION
**Pull request by keenanamigos**
Add timeIntervals object that contains the enum SECONDS for clarity and reusability

Add functions to parse and retreive the correct values for hours, minutes, and seconds within the convertTimestamp function for clarity and making code DRY

Change multiple single-line comment block to be a multi-line comment block

Create a keyCodes object that contains SPACEBAR as an enum to replace magic numbers in keyup and keydown event functions

My goal with this PR was to remove some of the repetitiveness I saw in the code and replace that with well named and reusable functions and variables. My hope is that these changes will give greater clarity to readers of the code and remove the need for some of the comments that were present.

Provide a general summary of your changes in the **Title** above.

### Important
Mark with [x] to select. Leave as [ ] to unselect.

### Motivation and Context
- [x] Why is this change required? What problem does it solve?
- [ ] If it fixes an open issue, include the text `Closes #1` (where 1 would be the issue number) to your commit message.

### Types of changes
What types of changes does your code introduce? Check all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
- [x] Describe your changes in detail.

### Final checklist:
Go over all the following points and check all the boxes that apply  
If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](.github/CONTRIBUTING.md) guidelines.
- [x] All tests passed.

### CODE REVIEW

File: js/everything.js [hunk 1]
```diff
@@ -498,17 +508,29 @@ function convertTimestamp(timestamp) {
...
 
+function convertHoursToSeconds(hours) {
+    return parseInt(hours, 10) * timeIntervals.SECONDS * timeIntervals.SECONDS;
+}
+
+function convertMinutesToSeconds(minutes) {
+    return parseInt(minutes, 10) * timeIntervals.SECONDS;
+}
+
+function getSeconds(seconds) {
```
Comment (shakeelmohamed) [inline js/everything.js]: I'd like to rename this function to something like `parseBase10Int` - and use it in the above 2 functions for code reuse.

File: js/everything.js [hunk 2]
```diff
@@ -223,11 +231,13 @@ var ZenPlayer = {
                 var videoDuration = plyrPlayer.plyr.embed.getDuration();
                 if (window.sessionStorage && videoDuration > 0) {
                     var currentTime = plyrPlayer.plyr.embed.getCurrentTime();
-                    // Only store the current time if the video isn't done
-                    // playing yet. If the video finished already, then it
-                    // should start off at the beginning next time.
-                    // There is a fuzzy 3 seconds because sometimes the video
-                    // will end a few seconds before the video duration.
+                    /*
+                        Only store the current time if the video isn't done
```
Comment (shakeelmohamed) [inline js/everything.js]: I'd prefer to see block comments in the following format:

```js
/**
  * Text starts here
  * continues here.
  */
```

**Comment by shakeelmohamed** [comment]
Thanks for the suggestions @keenanamigos, left you some comments

File: js/everything.js (hunk 1 above)
Comment (keenanamigos) [inline js/everything.js]: Great call, I like that idea a lot. I'll make the change.

**Pull request merged**